extensions/cql/standard/clause_6_filter.adoc

```diff
@@ -116,12 +123,16 @@ in this standard.
 === Parameter filter-crs
 
 For reasons discussed in the http://docs.opengeospatial.org/is/17-069r3/17-069r3.html#SDWBP[W3C/OGC Spatial Data on the Web Best Practices document], 
-<<OAFeat-1,OGC API - Features - Part 1: Core>> uses WGS 84 longitude and latitude and the default coordinate reference system. <<OAFeat-2,OGC API - Features - Part 2: Coordinate Reference Systems by Reference>> extends the capabilities of Part 1 to allow other coordinate reference systems to be used.  The `filter-crs` parameter defined in this clause allows clients to indicate to servers that support other CRSs, which CRS is being used to encode geometric values in a filter expression.
+<<OAFeat-1,OGC API - Features - Part 1: Core>> uses WGS 84 longitude and latitude (`http://www.opengis.net/def/crs/OGC/1.3/CRS84`) for 2-D coordinates and WGS 84 longitude, latitude and height in meters (`http://www.opengis.net/def/crs/OGC/0/CRS84h`) for 3-D coordinates as the default coordinate reference systems. <<OAFeat-2,OGC API - Features - Part 2: Coordinate Reference Systems by Reference>> extends the capabilities of Part 1 to allow other coordinate reference systems to be used.
```

Comment:
"... and height ..."  should be "... and ellipsoidal height ..." (see the recent editorial updates to part 1 and 2).